### PR TITLE
HW3 Pull Request

### PR DESCRIPTION
**Pull request by emmayakley**
## What I changed
Created a navbar with links to the list, population, and custom page and created these pages.
Also created an api folder to get the data from the api.
List page: lists the countries of South America, their flags, and information about each country.
Population page: displays a bar graph of the populations of each country.
Custom page: displays a pie graph of the number of countries that speak each language.

## What feedback I'm looking for
I had a hard time with the styling component of this homework, and ended up using CSS but not loving the outcome. I would love input on how you went about styling and any tips/feedback on what I could style better (I think most of it can be styled better). For the list page, I have everything listed vertically, but I think styling them in more of a card style would be better, but I ran out of time. Any other feedback is also welcome!

### CODE REVIEW

File: hw3/hw3/src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Link } from 'react-router-dom';
+import './Navbar.css';
+
+const Navbar = () => {
+  return (
+    <div>
```
Comment (mukundsai79) [inline hw3/hw3/src/components/Navbar.js]: you can remove <div> and directly use nav element

File: hw3/hw3/src/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React, {StrictMode} from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
```
Comment (mukundsai79) [inline hw3/hw3/src/index.js]: const root = ReactDOM.createRoot(document.getElementById('root'));

Create the root

File: hw3/hw3/src/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ReactDOM.render(
```
Comment (mukundsai79) [inline hw3/hw3/src/index.js]: root.render! (use render instead of ReactDOM.render())

File: hw3/hw3/src/index.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  <StrictMode>
```
Comment (mukundsai79) [inline hw3/hw3/src/index.js]: <React.StrictMode>

**Comment by turrilld** [comment]
Hi Emma!  As far as styling goes I think you did fine.  The graphs are clear, and that was the important bit.  My only real critique is that you could have used custom colors for your bar graph (there is an example of how to set that up in the starter project we were shown) As to the rest, I would recommend styling things with bootstrap first and then using CSS to restyle anything you dont like the look of.  Bootstrap comes with a lot of handy automatic styling that you dont need to override, which lets you focus on styling the elements that matter.  Beyond that, just choosing an interesting background color and a complementary text color goes a long way.  

As far as your code is concerned, it looks good.  Sai seems to have already caught everything I noticed and then some.  My only thing to add is that I see some commented out code in your population.js file on stackblitz, which you can probably remove if you are not going to come back to it.

**Pull request merged**